packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -715,10 +818,7 @@ contract OPContractsManager is ISemver {
 
     /// @notice Returns default, standard config arguments for the SystemConfig initializer.
     /// This is used by subclasses to reduce code duplication.
-    function defaultSystemConfigParams(
-        DeployInput memory, /* _input */
-        DeployOutput memory _output
-    )
+    function defaultSystemConfigParams(DeployOutput memory _output)
```

Comment:
We will keep the input argument even if unused because it avoids the need for interface changes for a child OPCM (e.g. OPCMInterop) that might need the `input` when encoding params